### PR TITLE
Resolved #2456: Exemption check bugs

### LINKED ISSUE #2456: ABAWD exemption finder bug: not calculating job income/hours correctly
If HH member has more than one job, the 1st job is skipped, and the 2nd - last is read.
Also prospective hours we're not being multiplied. The prospective hours were just being carried though. 

Fix is ready. Assign to me for the August release.

### PR DESCRIPTION
**Pull request by IlseFerris**
BLIP: Fixed a bug for cases in which HH member has more than one job. The 1st job was being skipped. All jobs now being read, and included in income calculation. 
Also fixed a bug to correctly multiply prospective hours for all jobs. 

This bug was fixed in all three scripts included in the pull request.

### CODE REVIEW

File: Script Files/BULK/BULK - BANKED MONTHS REPORT.vbs [hunk 1]
```diff
@@ -364,20 +367,20 @@ For item = 0 to UBound(Banked_Month_Client_Array, 2)		'Now each entry in the arr
...
+						prosp_hrs = prosp_hrs
 					Case "2"
-						prosp_hrs = prosp_hrs + pp_hrs * 2
+						prosp_hrs = (2 * prosp_hrs)
 					Case "3"
-						prosp_hrs = prosp_hrs + pp_hrs * 2.15
+						prosp_hrs = (2.15 * prosp_hrs)	
 					Case "4"
-						prosp_hrs = prosp_hrs + pp_hrs * 4.3
+						prosp_hrs = (2.15 * prosp_hrs)
```
Comment (CDPotter) [inline Script Files/BULK/BULK - BANKED MONTHS REPORT.vbs]: Can you change this to 4.3 * prosp_hrs for weekly?

File: Script Files/DAIL/DAIL - ABAWD FSET EXEMPTION CHECK.vbs [hunk 2]
```diff
@@ -245,17 +247,17 @@ FOR EACH person IN HH_member_array
 					prosp_inc = prosp_inc + prosp_monthly
 					EMReadScreen prosp_hrs, 8, 16, 50
 					IF prosp_hrs = "        " THEN prosp_hrs = 0
-					pp_hrs = pp_hrs * 1
```
Comment (CDPotter) [inline Script Files/DAIL/DAIL - ABAWD FSET EXEMPTION CHECK.vbs]: Why did you remove this?

File: Script Files/DAIL/DAIL - ABAWD FSET EXEMPTION CHECK.vbs [hunk 3]
```diff
@@ -267,23 +269,25 @@ FOR EACH person IN HH_member_array
 						prosp_inc = prosp_inc + prosp_monthly
 						EMReadScreen prosp_hrs, 8, 16, 50
 						IF prosp_hrs = "        " THEN prosp_hrs = 0
-						pp_hrs = pp_hrs * 1
```
Comment (CDPotter) [inline Script Files/DAIL/DAIL - ABAWD FSET EXEMPTION CHECK.vbs]: Why was this removed?

File: Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs [hunk 4]
```diff
@@ -302,23 +288,25 @@ FOR EACH person IN HH_member_array
 						prosp_inc = prosp_inc + prosp_monthly
 						EMReadScreen prosp_hrs, 8, 16, 50
 						IF prosp_hrs = "        " THEN prosp_hrs = 0
-						pp_hrs = pp_hrs * 1
```
Comment (CDPotter) [inline Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs]: Why was this removed?

File: Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs [hunk 5]
```diff
@@ -280,17 +266,17 @@ FOR EACH person IN HH_member_array
 					prosp_inc = prosp_inc + prosp_monthly
 					EMReadScreen prosp_hrs, 8, 16, 50
 					IF prosp_hrs = "        " THEN prosp_hrs = 0
-					pp_hrs = pp_hrs * 1
```
Comment (CDPotter) [inline Script Files/ACTIONS/ACTIONS - ABAWD FSET EXEMPTION CHECK.vbs]: Why was this removed?

**Comment by CDPotter** [comment]
Please resolve comments. I'm thinking we want the * 1 as it insures that the variable is a number able to be manipulated later.

**Comment by IlseFerris** [comment]
pp_hrs was removed as it didn't appear to serve any purpose, and it was removed as a variable in my updates and testing went without a hitch. Had it been commented initially and it's purpose made clear, I would not have removed it.|
I will add a similar variable to line up with my updates into the script to ensure that it reads as a numeric. Updates coming soon.

**Comment by theVKC** [comment]
On this note, @CDPotter and @IlseFerris: I'd like to investigate (or perhaps explain in documentation) why we use `* 1` as opposed to `abs` or `cint` (depending on context, obviously). Where there is a documented VBScript function, we ought to be using it.

**Comment by CDPotter** [comment]
I've always just done it that way because it seemed like every time I used CINT or ABS it would end up being wrong (causing type mismatches and what not) down the line somewhere. 

I think it could probably earn a spot in documentation.

**Comment by theVKC** [comment]
If it's wrong down the line though, doesn't that mean it's wrong for good reason? If we're manipulating strings and numerics, and transferring between the two, we ought to be training our scriptwriters to work with them correctly from-the-get-go.

**Comment by theVKC** [comment]
Essentially, if it's been converted it becomes the appropriate data type for our purposes, and if it causes a type mismatch it's because we're using it the wrong way later on (and, for sake of clean and neat code execution, should adjust it later on down the line). I know I train on `abs`, `cint`, and `FormatNumber`.

**Comment by theVKC** [comment]
Also, see this: http://www.w3schools.com/asp/func_cdbl.asp
